tests/kafkatest/services/performance/consumer_performance.py

```diff
@@ -78,6 +78,11 @@ def __init__(self, context, num_nodes, kafka, topic, messages, new_consumer=Fals
         self.new_consumer = new_consumer
         self.settings = settings
 
+        if version < V_0_9_0_0 and self.new_consumer:
+            self.logger.warn("Version is %s where new_consumer is not available, but new_consumer is set to True. "\
+                             "Overriding new_consumer flag to False..." % str(version))
+            self.new_consumer = False
```

Comment:
Looking back at this, it's a bit fishy - maybe better to enforce this with an assertion rather than warning and modifying the `self.new_consumer`